File: isa2/demo/src/main/java/isa2/demo/DTO/InstructorAvailabilityDTO.java
package isa2.demo.DTO;

import isa2.demo.Model.AvailabilityType;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class InstructorAvailabilityDTO {
    private Integer id;
    private PeriodDTO periodDTO;
    private Integer ownerId;
    private AvailabilityType availabilityType;
}

File: client/src/app/model/instructor-availability-dto.ts
import { AvailabilityType } from './../enum/availability-type.enum';
import { PeriodDTO } from './period-dto.model';

export class InstructorAvailabilityDto {
    public id!: number;
    public periodDTO!: PeriodDTO;
    public ownerId!: number;
    public availabilityType!: AvailabilityType;

}
